floris/tools/floris_interface.py

```diff
@@ -355,7 +355,7 @@ def _set_operation(
         """
         # Add operating conditions to the floris object
         if yaw_angles is not None:
-            self.floris.farm.yaw_angles = yaw_angles
+            self.floris.farm.yaw_angles = np.array(yaw_angles)
```

Comment:
Is this the only place where yaw angles are going to be set? If not, you might consider adding a setter method on `Farm` to do this always:

```python
class Farm:
    def set_yaw_angles(yaw_angles):
        # Do lots of checks
        ...
        # Cast to Numpy array
        self.yaw_angles = np.array(yaw_angles)
```

You could also use the `@property` decorator [the way we used to do](https://github.com/NREL/floris/blob/31fe1b69ff863f0a610aec5b22424382ec3cc933/floris/utilities.py#L96-L102), if you think maintaining the attribute-style use is important.

Reply:
Ah and I just read https://github.com/NREL/floris/pull/828#issuecomment-1979972559, sorry to say the same thing @misi9170

Reply:
No problem---I'm going to try to do this (use a setter method), it seems like a better approach